source/cli.js

```diff
@@ -75,6 +79,7 @@ updateNotifier({pkg: cli.pkg}).notify();
 	const defaultFlags = {
 		cleanup: true,
 		publish: true,
+		preview: false,
```

Comment:
This isn't necessary as just like the other flags that are `false` by default (`yolo`, `anyBranch`, etc.), so will `preview` be `false` by default.